db/migrate/20161007033805_create_teachers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTeachers < ActiveRecord::Migration
+  def change
+    create_table :teachers do |t|
+
+      t.boolean :is_searching, default: false
+      t.text :instruments, array: true, default: []
```

Comment:
can you add spaces after t.text so that :instruments matches up with :is_searching